### PR TITLE
Static OP & Base Token Updates

### PR DESCRIPTION
**Pull request by MSilb7**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by MSilb7** [comment]
seems like a prices config error? `ERROR:root:ERROR: chain: base not supported in the price checker, could not check contract or chain`

File: models/prices/base/prices_base_tokens.sql [hunk 1]
```diff
@@ -18,7 +18,14 @@ FROM
...
 --    ('weth-weth', 'base', 'WETH', 0x4200000000000000000000000000000000000006, 18), --requested add to coinPaprika 2023-08-10
 --    ('axl-axelar', 'base', 'AXL', 0x467719aD09025FcC6cF6F8311755809d45a5E5f3, 6), --requested add to coinPaprika 2023-08-10
-    ('bald-bald', 'base', 'BALD', 0x27D2DECb4bFC9C76F0309b8E88dec3a601Fe25a8, 18)
+    ('bald-bald', 'base', 'BALD', 0x27D2DECb4bFC9C76F0309b8E88dec3a601Fe25a8, 18),
+    ('usdbc-usd-base-coin', 'base', 'USDbC', 0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA, 6),
+    ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
+    ('weth-weth', 'base', 'WETH', 0x4200000000000000000000000000000000000006, 18),
+    ('dai-dai', 'base', 'DAI', 0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb, 18),
+    ('cbeth-coinbase-wrapped-staked-eth', 'base', 'cbETH', 0x2Ae3F1Ec7F1F5012CFEab0185bfc7aa3cf0DEc22, 18)
+
```
Comment (Hosuke) [inline models/prices/base/prices_base_tokens.sql:27]: Currently price feeds for tokens on base are still not supported by coinspaprika:
https://api.coinpaprika.com/v1/contracts/

Comment (MSilb7) [reply models/prices/base/prices_base_tokens.sql:27]: Can we just use the existing price feeds?

Like take the cbeth price feed and apply it to Base?

Comment (Hosuke) [reply models/prices/base/prices_base_tokens.sql:27]: Yes we can. If the token existed in other chains.

Comment (Hosuke) [reply models/prices/base/prices_base_tokens.sql:27]: In this case we may need to manually check the API.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @MSilb7

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**